node_modules/oae-core/editprofile/js/editprofile.js

```diff
@@ -111,9 +111,29 @@ define(['jquery', 'oae.core'], function ($, oae) {
             }
 
             // Render the form elements
+            // detect if is possible to modify the name and email
+            var disableModification = false;
+            var EnableModification = true;
```

Comment:
We should only need one of these variables since they will always be opposites (if `disableModification` if false `enableModification` will always be true).